cli/pkg/kctrl/cmd/package/installed/create_or_update.go

```diff
@@ -274,8 +291,9 @@ func (o *CreateOrUpdateOptions) RunUpdate(args []string) error {
 		return fmt.Errorf("Expected package install to be non-empty")
 	}
 
-	if len(o.version) == 0 && len(o.valuesFile) == 0 && o.values {
-		return fmt.Errorf("Expected either package version or values file to update the package")
+	if len(o.version) == 0 && len(o.valuesFile) == 0 && o.values &&
+		o.YttOverlayFlags.yttOverlays && len(o.YttOverlayFlags.yttOverlayFiles) == 0 {
```

Comment:
It feels like this is getting more and more complex. Should we break this into bits? Something like:
```go
hasValuesFile := o.values && len(o.valuesFile) > 0
hasYttOverlayFiles := o.YttOverlayFlags.yttOverlays && len(o.YttOverlayFlags.yttOverlayFiles) > 0

if len(o.version) == 0 && !hasValuesFile && !hasYttOverlayFiles {}
```

Reply:
I was planning on leavinga comment instead

Reply:
I think the switch case method Dmitriy suggested would help out?